color.go

```diff
@@ -321,38 +322,37 @@ func (c *Color) PrintlnFunc() func(a ...interface{}) {
 //	put := New(FgYellow).SprintFunc()
 //	fmt.Fprintf(color.Output, "This is a %s", put("warning"))
 func (c *Color) SprintFunc() func(a ...interface{}) string {
-	return func(a ...interface{}) string {
-		return c.wrap(fmt.Sprint(a...))
-	}
+	return c.Sprint
 }
 
 // SprintfFunc returns a new function that returns colorized strings for the
 // given arguments with fmt.Sprintf(). Useful to put into or mix into other
 // string. Windows users should use this in conjunction with color.Output.
 func (c *Color) SprintfFunc() func(format string, a ...interface{}) string {
-	return func(format string, a ...interface{}) string {
-		return c.wrap(fmt.Sprintf(format, a...))
-	}
+	return c.Sprintf
 }
 
 // SprintlnFunc returns a new function that returns colorized strings for the
 // given arguments with fmt.Sprintln(). Useful to put into or mix into other
 // string. Windows users should use this in conjunction with color.Output.
 func (c *Color) SprintlnFunc() func(a ...interface{}) string {
-	return func(a ...interface{}) string {
-		return c.wrap(fmt.Sprintln(a...))
-	}
+	return c.Sprintln
 }
 
 // sequence returns a formatted SGR sequence to be plugged into a "\x1b[...m"
 // an example output might be: "1;36" -> bold cyan
 func (c *Color) sequence() string {
-	format := make([]string, len(c.params))
+	if len(c.params) == 0 {
+		return ""
+	}
+	var sb strings.Builder
```

Comment:
Any reason for the switch to a string builder? is it more efficient that the array Join?

Reply:
Yes, it's faster. `Join` uses `strings.Builder` under the hood, by using it directly, we can avoid creating the slice in the first place.